### DISCUSSION TITLE
the code snippits didnt change on the totorials

### DISCUSSION THREAD

**Discussion by Tizum5** [feathersjs/feathers#3174] [Q&A]
### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/dove/.github/contributing.md#report-a-bug) - if it is a general question or suggestion please start a [Discussion](https://github.com/feathersjs/feathers/discussions))

- [the code snippets didnt change to js after you first download feathers] Tell us what broke. The more detailed the better.
- [https://feathersjs.com/guides/basics/starting.html ] If you can, please create a simple example that reproduces the issue and link to a gist, jrepo, etc. This makes it much easier for us to debug and issues that have a reproducible example will get higher priority.

### Expected behavior

Tell us what should happen

### Actual behavior

Tell us what happens instead

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:

**Operating System**:

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by daffl** [comment]
Can you provide more information? Operating system, web browser and version etc.
